x-pack/plugins/observability_solution/observability_onboarding/public/application/quickstart_flows/system_logs/install_elastic_agent.tsx

```diff
@@ -118,7 +120,9 @@ export function InstallElasticAgent() {
         });
       }
     },
-    [apiKeyEncoded, onboardingId, installShipperSetupStatus === FETCH_STATUS.SUCCESS]
+    // FIXME: Dario could not find a reasonable fix for successfullyInstalledShipperSetup
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [apiKeyEncoded, onboardingId, successfullyInstalledShipperSetup]
```

Comment:
wouldn't it stop complaining if you add `successfullyInstalledShipperSetup` to the `if` inside this useFetcher?

Reply:
nah, it will then complain that it's not in the list of dependencies - it's a behavioral change as well which I would like to avoid to some extent.